### DISCUSSION TITLE
Any up-to-date GUI client?

### DISCUSSION THREAD

**Discussion by FiftyFour7250** [ScoopInstaller/Scoop#4660] [General]
Hey guys. Is there any up-to-date GUI Scoop client? I found [scoop-viewer](https://github.com/prezesp/scoop-viewer), but the last update was in May 2020(!), and there is not even an option to update installed packages. May be there is something like Chocolatey GUI?

**Comment by twilightsophie** [comment]
This is a GUI for Winget and Scoop, it supports extras buckets too - actually found it worked better for Scoop apps than Winget apps. You can click on the app for information and links to the website (the stuff Scoop shows if you put in info) and that's where the Install button is. [WinGetUI](https://github.com/martinet101/WinGetUI).

**Comment by rashil2000** [comment]
There is another one - https://github.com/OzakIOne/scoop-gui, but you need to build it
